_clients/data-prepper/pipelines.md

```diff
@@ -204,6 +204,49 @@ metrics-pipeline:
       password: admin
 ```
 
+### S3 Log Ingestion Pipeline
+
+The following example demonstrates how to use the S3 source and Grok prepper plugins to process unstructured log data
+from S3. This example uses Amazon Application Load Balancer (ALB) logs. As the ALB writes logs to S3, S3 creates notifications
```

Comment:
Change "Amazon Application Load Balancer (ALB)" to "Application Load Balancer". In the second sentence, change "ALB" to "Application Load Balancer" (do not define this as "ALB").